src/sage/schemes/toric/weierstrass.py

```diff
@@ -286,7 +286,7 @@ def Newton_polytope_vars_coeffs(polynomial, variables):
         e = m.exponents()[0]
         v = tuple([e[i] for i in var_indices])
         m_red = m // prod(x**i for x, i in zip(variables, v))
-        result[v] = result.get(v, R.zero()) + c*m_red
+        result[v] = result.get(v, R.zero()) + c * m_red
```

Comment:
```suggestion
        result[v] = result.get(v, R.zero()) + c*m_red
```
whitespace around top-level arith operators only please